recipes/xmlsec/all/test_package/rsakey.pem

```diff
@@ -0,0 +1,9 @@
+-----BEGIN RSA PRIVATE KEY-----
+MIIBPAIBAAJBANPQbQ92nlbeg1Q5JNHSO1Yey46nZ7GJltLWw1ccSvp7pnvmfUm+

```

Comment:
@steinerthomas 
I hope this isn't the private ssh key of your system 😛

Reply:
Oops... ;)
Who will merge this pr? I guess we wait for all the reviewers and anyone from @conan-io will merge it?
Thanks again for your support. I liked, that it was really uncomplicated to work together with all of you!

Reply:
See https://github.com/conan-io/conan-center-index/issues/2857 for the current process.
It should get merged anytime soon as you have received sufficient approvals.

Reply:
bot merged it. 👍  (Enough reviews)

Reply:
yes, exactly